CMakeLists.txt

```diff
@@ -21,13 +21,6 @@ cmake_minimum_required(VERSION 3.18.2)
 project(skymp)
 enable_testing()
 
-# Bool options
-option(JS_ENGINE_TRACING_ENABLED "Useful for JsEngine debugging. See JsEngine.h" OFF)
```

Comment:
Idk if that's a good idea, but i don't feel like this cmake opt should be completely removed. I feel like it can be still useful to specify it using cmake instead of editing the source code